client/src/app/site/pages/meetings/pages/motions/services/list/motion-list-filter.service/motion-list-filter.service.ts

```diff
@@ -240,7 +261,8 @@ export class MotionListFilterService extends BaseMeetingFilterListService<ViewMo
             this.motionBlockFilterOptions,
             this.recommendationFilterOptions,
             this.motionCommentFilterOptions,
-            this.tagFilterOptions
+            this.tagFilterOptions,
+            ...this.forwardingFilterOptions
```

Comment:
Why do you use the `...` operator here and not for the other options? `forwardingFilterOptions` returns only one item, anyways.

Reply:
Ah yeah, that's a relic from when I  still didn't know whether I would create two filters.
I'll change that